tools/nbs_tools/decode_image/video_recorder.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import os
+import queue
+import threading
+from subprocess import DEVNULL, PIPE, Popen, call
+
+
+class Recorder:
+    def __init__(self, output_path, camera_name, dimensions, pixel_format, encoder, buffer_size=100):
+
+        self.timecode_path = os.path.join(output_path, "{}_timecode.txt".format(camera_name))
+        self.video_path = os.path.join(output_path, "{}.mp4".format(camera_name))
+        self.buffer_size = buffer_size
+
+        self.timecode = open(self.timecode_path, "w")
+        self.frames = []
+        self.start_time = None
+
+        self.encoder = Popen(
+            [
+                "ffmpeg",
```

Comment:
Has ffmpeg been added to the docker image? Or has this dependency been introduced in some other manner?

Reply:
this runs on the host, not on docker, so presumably, you'd need to have FFmpeg installed to run `./b nbs video`